src/main/java/seedu/address/model/wedding/Wedding.java

```diff
@@ -0,0 +1,25 @@
+package seedu.address.model.wedding;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import seedu.address.model.person.Person;
+
+public class Wedding {
+    //TODO: create class for WeddingName
+    private final String name;
+    private final Set<Person> contacts = new HashSet<>();
+
+    public Wedding(String name, Set<Person> contacts) {
+        this.name = name;
+        this.contacts.addAll(contacts);
+    }
```

Comment:
I personally think there should be another field for the clients (wedding couple), and contacts could be further clarified to be serviceProviders. With a couple field it would be easier to distinguish different weddings from each other (weddings are not duplicate as long as they contain different couples) and  would simplify the access of the couple's contact information. This way, you wouldn't need to additionally filter through all contacts to find the clients/couple.

Also, I would suggest the addition of a weddingDate field and getDate/setDate methods. Besides that, everything else LGTM!!